zipkin-web/package.json

```diff
@@ -20,7 +20,8 @@
     "flightjs": "^1.5.1",
     "jquery": "^2.2.0",
     "js-cookie": "^2.1.0",
-    "moment": "^2.11.1",
+    "lodash": "^4.5.0",
+    "moment": "^2.11.2",
```

Comment:
lodash is great for doing functional stuff on collections, mimicking the Scala implementation which uses Scala's collection API a lot.